reference/content-types/resource_locator.rst

```diff
@@ -19,10 +19,11 @@ Tags
     * - Tag
       - Description
     * - sulu.rlp
-      - The resource locator with this tag defines the URL to a specific page
+      - The resource locator with this tag defines the URL to a specific page.
     * - sulu.rlp.part
-      - Fields marked with this tag are used to generate the URL for a specific
-        page
+      - Fields marked with this tag are used to generate the URL for a specific page.
+        If more than one field ist marked, the values of these fields will be concatenated into the resource locator.
+        The sequence depends on the value of the priority parameter.
```

Comment:
Sadly that's not true, this is a bug at the moment 😕 I think as long as it is not working we should not have it like this in the documentation.